pygeoapi/api.py

```diff
@@ -818,11 +1028,47 @@ def conformance(self,
 
         return headers, 200, to_json(conformance, self.pretty_print)
 
+    @gzip
+    @pre_process
+    @jsonldify
+    def get_describe_collections(self, request: Union[APIRequest, Any],
+                                 dataset=None) -> Tuple[dict, int, str]:
+        """
+        Provide collection metadata
+
+        :param request: A request object
+        :param dataset: name of collection
+
+        :returns: tuple of headers, status code, content
+        """
+
+        # Redirect to the common describe_collections method
+        # specifying the GET method was used
+        return self.describe_collections(request, dataset, "GET")
```

Comment:
Why isn't the method addicted to the `APIRequest`?

Reply:
This can be discussed further. My take is that `self.describe_collections` here refers to `API` logic code, reminiscent of the other get/post handlers for the other endpoints. I think it applies to leave it in `API` class rather than `APIRequest`. To me, `APIRequest` would only have utility methods to handle specifically the request parameters, not so much as the API core logic.